source/vsc_tutorials.rst

```diff
@@ -5,14 +5,24 @@ VSC tutorials
 Site Tutorials
 ==============
 
-VSC sites carry out regular trainings about Linux and the VSC HPC systems. It is
-complementary to the information found in this user portal, the latter being
-the reference manual. Below you can find links to those training materials.
+VSC sites carry out regular trainings about Linux and the VSC HPC systems. 
+You may always find the updated list of `VSC trainings <https://www.vscentrum.be/vsctraining>`__ 
+via the VSC website. It is complementary to the information found in this 
+user portal, the latter being the reference manual. Below you can find links 
+to those training materials.
 
 KULeuven/UHasselt
 -----------------
 
+The training material (slides and exercises) for the a variety of HPC-related trainings from
+the KULeuven/UHasselt site are organized via the 
+`official Github repository <https://github.com/hpcleuven>`__. 
+The beginners might consider following:
```

Comment:
```suggestion

New users without prior experience with Linux or the HPC might consider the following introductory tutorials: 
```

Reply:
Thanks @lexming . This is improved in `08efa84`